tests/v2/integration/projects/resource_quota_test.go

```diff
@@ -1,18 +1,29 @@
 package integration
 
 import (
+	"context"
+	"fmt"
 	"testing"
-	"time"
 
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
 	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/resourcequotas"
 	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
-	"github.com/rancher/rancher/tests/framework/extensions/resourcequotas"
+	steveResourceQuotas "github.com/rancher/rancher/tests/framework/extensions/resourcequotas"
 	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
 	"github.com/stretchr/testify/suite"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	resourcdQuotaNamespaceName  = "ns1"
+	resourcdQuotaNamespaceName2 = "ns2"
+	localClusterID              = "local"
```

Comment:
This constant can also be used for project config structs that are created within the tests, it might be easier to change them all if that happens, what do you all think? I don't have a strong opinion on this

Reply:
You're right, but since this test is only in v2.7. Let's just keep this like this for now.